website/docs/components/tabs/partials/guidelines/guidelines.md

```diff
@@ -47,6 +47,20 @@ Don’t trigger a Tab change via an external action, such as a submit or next bu
 ![Tabs with external trigger](/assets/components/tabs/tabs-external-trigger.png)
 !!!
 
+## Sizes
+
+Medium-sized tabs work best in most contexts. Large tabs can be used when needed to emphasize important sections, enhance clarity, and convey hierarchical distinction. 
+
+![Tabs sizes example](/assets/components/tabs/tab-size.png)
+
+### Choosing sizes 
+
+Mixing different tab sizes is not recommended, as it can confuse users. Choose tab sizes with a purpose, aligning them with the hierarchical importance of the information they represent. 
+
+### Organizing 
+
+We recommend to structure your tabs to reflect the hierarchical organization. Large tabs are reserved for major differences, emphasizing their importance, and should be used sparingly. Medium tabs, by contrast, are more adaptable and suited for grouping content under those significant categories, promoting an organized and straightforward experience within specific sections or among related content.
```

Comment:
```suggestion
We recommend structuring Tabs to reflect the intended hierarchy of the content they contain. Large tabs are reserved for major differences, emphasizing their importance, and should be used sparingly. Medium tabs, by contrast, are more adaptable and suited for grouping content under those significant categories, promoting an organized and straightforward experience within specific sections or among related content.
```

It actually might be reasonable to break these into bullets (kind of what we talked about today), maybe consider this:

We recommend structuring Tabs to reflect the intended hierarchy of the content they contain:

- Large Tabs sit higher in the page hierarchy and can be used to express more important content, but should be used more sparingly.
- Medium tabs are more adaptable in most scenarios and naturally occupy the same hierarchy as most page content.

Reply:
I think the suggested refactor using bulleted points is much clearer.